  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00,  3.63it/s]100%|██████████| 1/1 [00:00<00:00,  3.63it/s]
0it [00:00, ?it/s]ns before calling NumCudaDevices() that might have already set an error? Error 804: forward compatibility was attempted on non supported HW (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729138878/work/c10/cuda/CUDAFunctions.cpp:100.)
  return torch._C._cuda_getDeviceCount() > 0
Traceback (most recent call last):
  File "bm.eval.py", line 25, in <module>
    nvidia_smi.nvmlInit()
  File "/opt/miniconda3/envs/pialnn/lib/python3.7/site-packages/pynvml.py", line 615, in nvmlInit
    _nvmlCheckReturn(ret)
  File "/opt/miniconda3/envs/pialnn/lib/python3.7/site-packages/pynvml.py", line 310, in _nvmlCheckReturn
    raise NVMLError(ret)
pynvml.NVMLError_LibRmVersionMismatch: RM has detected an NVML/RM version mismatch.
  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00,  3.59it/s]100%|██████████| 1/1 [00:00<00:00,  3.59it/s]
0it [00:00, ?it/s]1it [00:03,  3.45s/it]1it [00:03,  3.45s/it]
1it [00:03,  3.50s/it]1it [00:03,  3.50s/it]
